src/components/BounceRules/BounceRules.css

```diff
@@ -0,0 +1,3 @@
+.bounceRules-ruleContainer:hover {
```

Comment:
can you use .scss instead of .css to keep it consistent with what we use at SendGrid?